### DISCUSSION TITLE
Open termux app and run command without user interaction

### DISCUSSION THREAD

**Discussion by Coder-ShivamShrivastava** [termux/termux-app#3648] [General]
I want to open termux app through intent (In android) and run command without user input or interaction. Directly launch app and run command.
Is it possible?

Thanks in Advanced.

**Comment by twaik** [comment]
You can simply put your command to `~/.bashrc` file... In the case if you are using Java to send intents you can use [`RUN_COMMAND` intent mechanism](https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent).

**Reply by Coder-ShivamShrivastava** [reply]
i will really grateful, if it is possible to you, can you attach code snippet ?? or explain in detail because i am able to open termux app through intent but unable to execute the command which are pass by intent ( intent.putExtra("com.termux.app.execute", arrayOf<String>("-c","ls")) )

**Reply by twaik** [reply]
https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent#advance-examples

**Reply by Coder-ShivamShrivastava** [reply]
Thank You
